static/scripts/pay.ts

```diff
@@ -16,10 +16,19 @@ const ErrorHandler = (error: any, extra: string | undefined = undefined) => {
 };
 
 const connectWallet = async (): Promise<JsonRpcSigner> => {
-  const provider = new ethers.providers.Web3Provider((window as any).ethereum, "any");
-  await provider.send("eth_requestAccounts", []);
-  const signer = provider.getSigner();
-  return signer;
+  try {
+    const provider = new ethers.providers.Web3Provider((window as any).ethereum, "any");
+    await provider.send("eth_requestAccounts", []);
+    const signer = provider.getSigner();
+    return signer;
+  } catch (error: any) {
+    if(error?.message?.includes("missing provider")) {
+      alert("Error: Please use a web3 enabled browser.");
+    } else {
+      alert("Error: Please connect your wallet.");
```

Comment:
Please don't use alert. Instead, this should just error in the console.

Reply:
> Please don't use alert. Instead, this should just error in the console.

Oh, i just thought they should see the error clearly

Reply:
There is another active pull request which displays errors in a toaster which we should seek to integrate here once that's merged in, optimally.